source/server/linux/syslog_logging.cpp

```diff
@@ -3,7 +3,8 @@
 #include <syslog.h>
 
 namespace nidevice_grpc {
-namespace Logging {
+
```

Comment:
We tend not to have blank lines between our namespaces. Same with the `}`s at the end.